config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/DependentBundle.java

```diff
@@ -0,0 +1,61 @@
+package com.ca.apim.gateway.cagatewayconfig.beans;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+import java.io.File;
+@JsonPropertyOrder({"type", "name", "groupName", "version"})
+public class DependentBundle {
```

Comment:
Can you add `implement Metadata` to it?

Reply:
@nikhilahuja we dont have id and guid for this